Java/core/src/test/java/com/amazon/randomcutforest/store/LeafStoreTest.java

```diff
@@ -24,12 +24,12 @@
 public class LeafStoreTest {
```

Comment:
We should rewrite this test and `NodeStoreTest` to cover both small and regular stores. We can create an issue to do that in a follow up PR if you don't want to add it here.

Reply:
Yes, lets keep that separate. Since multiple things have changed, we would perhaps have to look at overall test coverage as well.